core/src/test/java/org/springframework/security/authentication/DelegatingReactiveAuthenticationManagerTests.java

```diff
@@ -77,4 +77,36 @@ public void authenticateWhenBadCredentialsThenDelegate2NotInvokedAndError() {
 			.verify();
 	}
 
+	@Test
+	public void authenticateWhenDelegate1BadCredentialsThenDelegate2Authenticate() {
+		given(this.delegate1.authenticate(any())).willReturn(Mono.error(new BadCredentialsException("Test")));
+		given(this.delegate2.authenticate(any())).willReturn(Mono.just(this.authentication));
+
+		DelegatingReactiveAuthenticationManager manager = new DelegatingReactiveAuthenticationManager(true,
+				this.delegate1, this.delegate2);
+
+		assertThat(manager.authenticate(this.authentication).block()).isEqualTo(this.authentication);
+	}
+
+	@Test
+	public void authenticateWhenBothDelegatesBadCredentialsThenError() {
+		given(this.delegate1.authenticate(any())).willReturn(Mono.error(new BadCredentialsException("Test")));
+		given(this.delegate2.authenticate(any())).willReturn(Mono.error(new BadCredentialsException("Test")));
+
+		DelegatingReactiveAuthenticationManager manager = new DelegatingReactiveAuthenticationManager(true,
+				this.delegate1, this.delegate2);
+
+		StepVerifier.create(manager.authenticate(this.authentication))
+				.expectError(BadCredentialsException.class)
+				.verify();
+	}
+	@Test
```

Comment:
Please add some whitespace between the two tests here.

Reply:
Done.